Formula/consul.rb

```diff
@@ -14,13 +14,10 @@ class Consul < Formula
     sha256 "1b34a79b804fc7a7fa6060f7b1467f366ca8ed4d95265caee93acd52487ec671" => :high_sierra
   end
 
-  depends_on "go@1.12" => :build
+  depends_on "go" => :build
   depends_on "gox" => :build
 
   def install
-    # Avoid running `go get`
-    inreplace "GNUmakefile", "go get -u -v $(GOTOOLS)", ""
-
     ENV["XC_OS"] = "darwin"
     ENV["XC_ARCH"] = "amd64"
```

Comment:
Are these still needed?

Reply:
These two are needed, otherwise all platforms would be built.
https://github.com/hashicorp/consul/blob/master/build-support/functions/20-build.sh#L243-L247